litestar/types/serialization.py

```diff
@@ -55,8 +64,6 @@
     "IPv4Address | IPv4Interface | IPv4Network | IPv6Address | IPv6Interface | IPv6Network"
 )
 EncodableMsgSpecType: TypeAlias = "Ext | Raw | Struct"
-EncodablePydanticType: TypeAlias = (
-    "BaseModel | ByteSize | ConstrainedBytes | ConstrainedDate | NameEmail | SecretField | StrictBool | Color"
-)
+EncodablePydanticType: TypeAlias = "BaseModel | ByteSize | ConstrainedBytes | ConstrainedDate | NameEmail | SecretField | StrictBool | Color"  # type: ignore
```

Comment:
Should `UUID` also be here?  Does v2  use `SecretField` instead of `SecretStr` and `SecretBytes`?

Reply:
No, UUID is a builtin. SecretSTR and SecretBytes are subclasses of the respective types and as such encodable.